index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>MarkEllisDev Portfolio</title>
+
+	<meta charset="utf-8">
+	<link rel="stylesheet" type="text/css" href="css/styles.css">
+
+</head>
+<body>
+	<header>
+		<h1>Mark Ellis - Developer Portfolio</h1>
+	</header>
+
+	<article>
+		<h2>Bio</h2>
+		<img id="profilePic" src="images/mongoose_smoking.jpg" alt="Mark Ellis profile pic">
+		<p>This will eventually be my bio. Right now it's a placeholder.</p>
+
+	</article>
+
+	<article>
+		<table class=tableLayout3>
+			
+
+	 This will contain sites and links to sites I like, but for now, it contains my two older sites.
```

Comment:
This line should be inside a tag. Maybe a table header or something. It is not valid html.